### PR TITLE
[SPARK-17449] [Documentation] [Relation between heartbeatInterval and…

### PR DESCRIPTION
**Pull request by jagadeesanas2**
## What changes were proposed in this pull request?

The relation between spark.network.timeout and spark.executor.heartbeatInterval should be mentioned in the document. 

… network timeout]

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala [hunk 1]
```diff
@@ -32,6 +32,8 @@ import org.apache.spark.util._
  * A heartbeat from executors to the driver. This is a shared message used by several internal
  * components to convey liveness or execution information for in-progress tasks. It will also
  * expire the hosts that have not heartbeated for more than spark.network.timeout.
+ * spark.executor.heartbeatInterval configuration value is certainly intended to be much
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala]: OK, the verbatim quote from my email is a little funny with 'certainly', but, not a big deal.

Comment (jagadeesanas2) [inline core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala]: Thanks for correcting me..!! I will edit and send PR once again.

**Comment by srowen** [comment]
I think you can/should add the note to the user-facing configuration doc too. Same note.

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**